templates/terraform/examples/scan_config_basic.tf.erb

```diff
@@ -0,0 +1,16 @@
+resource "google_compute_address" "scanner_static_ip" {
+  provider = "google-beta"
+  name     = "<%= ctx[:vars]['address_name'] %>"
+}
+
+resource "google_security_scanner_scan_config" "<%= ctx[:primary_resource_id] %>" {
+	provider                          = "google-beta"
```

Comment:
nit: looks like you accidentally got some tabs in here. also no need to have quite so many spaces between the keys and values (terraform fmt will typically put one space after the longest key and then align the `=`s with each other)